### PR TITLE
Add documentation for asset saved reports and cloud cost reports

### PR DESCRIPTION
**Pull request by avrodrigues5**
## Related Issue

https://kubecost.atlassian.net/browse/ENG-2510

## Proposed Changes
* Added filter options available in cloud cost API
* Document Asset saved and cloud cost saved reports

### CODE REVIEW

**Review by srpomeroy — CHANGES_REQUESTED** [review]
Do we support `chartDisplay` for Assets and Cloud Costs?
What about `stepSize` for Allocations and Assets?

We call out "_Resolution_ in the UI" in a few places. Does this mean step size?

File: apis/filters-api.md [hunk 1]
```diff
@@ -57,6 +57,17 @@ v2 filter support for `/model/assets`:
 * `service`
 * `label`
 
+### Cloud Costs API
+
+v2 filter support for `/model/cloudCost`:
+
+* `accountID`
```
Comment (srpomeroy) [inline apis/filters-api.md:64]: Category too?

```suggestion
* `accountID`
* `category`
```

File: using-kubecost/navigating-the-kubecost-ui/reports.md [hunk 2]
```diff
@@ -83,11 +83,74 @@ The following fields apply to each map item under the `reports` key:
...
+  * number of days: `{N}d` (last N days)
+    * e.g. `30d` for the last 30 days
+  * date range: `{start},{end}` (comma-separated RFC-3339 date strings or Unix timestamps)
+    * e.g. `2021-01-01T00:00:00Z,2021-01-02T00:00:00Z` for the single day of 1 January 2021
+    * e.g. `1609459200,1609545600` for the single day of 1 January 2021
+  * _Note: for all window options, if a window is requested that spans "partial" days, the window will be rounded up to include the nearest full date(s)._
+    * e.g. `2021-01-01T15:04:05Z,2021-01-02T20:21:22Z` will return the two full days of 1 January 2021 and 2 January 2021
+* `aggregateBy` the desired aggregation parameter -- equivalent to _Breakdown_ in the Kubecost Asset UI. Supports:
+  * `account`
+  * `cluster`
```
Comment (srpomeroy) [inline using-kubecost/navigating-the-kubecost-ui/reports.md:111]:
```suggestion
  * `cluster`
  * `label` requires the following format: `label:<label_name>`
```

File: using-kubecost/navigating-the-kubecost-ui/reports.md [hunk 3]
```diff
@@ -83,11 +83,74 @@ The following fields apply to each map item under the `reports` key:
...
+  * number of days: `{N}d` (last N days)
+    * e.g. `30d` for the last 30 days
+  * date range: `{start},{end}` (comma-separated RFC-3339 date strings or Unix timestamps)
+    * e.g. `2021-01-01T00:00:00Z,2021-01-02T00:00:00Z` for the single day of 1 January 2021
+    * e.g. `1609459200,1609545600` for the single day of 1 January 2021
+  * _Note: for all window options, if a window is requested that spans "partial" days, the window will be rounded up to include the nearest full date(s)._
+    * e.g. `2021-01-01T15:04:05Z,2021-01-02T20:21:22Z` will return the two full days of 1 January 2021 and 2 January 2021
+* `aggregateBy` the desired aggregation parameter -- equivalent to _Breakdown_ in the Kubecost Cloud Cost UI. Supports:
+  * `accountID`
+  * `provider`
```
Comment (srpomeroy) [inline using-kubecost/navigating-the-kubecost-ui/reports.md:142]:
```suggestion
  * `label` requires the following format: `label:<label_name>`
  * `provider`
```

File: using-kubecost/navigating-the-kubecost-ui/reports.md [hunk 4]
```diff
@@ -83,11 +83,74 @@ The following fields apply to each map item under the `reports` key:
...
+  * keywords: `today`, `week` (week-to-date), `month` (month-to-date), `yesterday`, `lastweek`, `lastmonth`
+  * number of days: `{N}d` (last N days)
+    * e.g. `30d` for the last 30 days
+  * date range: `{start},{end}` (comma-separated RFC-3339 date strings or Unix timestamps)
+    * e.g. `2021-01-01T00:00:00Z,2021-01-02T00:00:00Z` for the single day of 1 January 2021
+    * e.g. `1609459200,1609545600` for the single day of 1 January 2021
+  * _Note: for all window options, if a window is requested that spans "partial" days, the window will be rounded up to include the nearest full date(s)._
+    * e.g. `2021-01-01T15:04:05Z,2021-01-02T20:21:22Z` will return the two full days of 1 January 2021 and 2 January 2021
+* `aggregateBy` the desired aggregation parameter -- equivalent to _Breakdown_ in the Kubecost Cloud Cost UI. Supports:
+  * `accountID`
```
[the remark this answers is not in the document]
Comment (avrodrigues5) [reply using-kubecost/navigating-the-kubecost-ui/reports.md:142]: Category validation was failing in Backend. Fixed it via PR https://github.com/kubecost/kubecost-cost-model/pull/2620. Merging the suggestion as category aggregation will work!

**Comment by avrodrigues5** [comment]
> Do we support `chartDisplay` for Assets and Cloud Costs? What about `stepSize` for Allocations and Assets?
>  At this time No

> We call out "_Resolution_ in the UI" in a few places. Does this mean step size?

Yes, it's right. Im keeping the same used name for both but it means step size.

**Review by srpomeroy — APPROVED** [review]

**Pull request merged**